test/basic.bench.ts

```diff
@@ -1,4 +1,4 @@
-import { benchmarkSuite } from "../dist";
+import { benchmarkSuite } from "../src";
```

Comment:
Please change this back. Import from dist more accurately present usage of this library.